soccer/MainWindow.cpp

```diff
@@ -689,8 +727,8 @@ void MainWindow::updateViews() {
             }
 
             // check for kicker error code
-            bool kickerFault =
-                rx.has_kicker_status() && (rx.kicker_status() & 0x80);
+            bool kickerFault = rx.has_kicker_status() &&
+                               !(rx.kicker_status() & Kicker_Enabled);
```

Comment:
good catch, I *think* so. I do know these values work though, so I think there's a change on the firmware side. @jpfeltracco, @petersonev, was there a change that coresponds to this one in firmware?